.goreleaser.yaml

```diff
@@ -0,0 +1,62 @@
+version: 2
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - id: jimmsrv
+    main: ./cmd/jimmsrv
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    flags:
+      - -trimpath
+    ldflags:
+      - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      # - darwin
+      # - windows
+    goarch:
+      - amd64
+      # - arm64
+    ignore:
+    # TODO: disabled => juju/utils symlink_windows.go:54:8: undefined: createSymbolicLink
+    - goos: windows
+      goarch: arm64
+    binary: 'jimmsrv_v{{ .Version }}'
+
+  - id: jaas
+    main: ./cmd/jaas
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    flags:
+      - -trimpath
+    ldflags:
+      - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      # - darwin
+      # - windows
+    goarch:
+      - amd64
+      # - arm64
+    ignore:
+    # TODO: disabled => juju/utils symlink_windows.go:54:8: undefined: createSymbolicLink
+    - goos: windows
+      goarch: arm64
+    binary: 'jaas_v{{ .Version }}'
+
+archives:
+  - formats: [tar.gz]
+    # this name template makes the OS and Arch compatible with the results of `uname`.
+    name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
```

Comment:
Do we need a special format? I guess it is because we're putting both binaries into a single archive so we need to provide a name?